Agents/ForecastingAgent/forecasting/datamodel/data_mapping.py

```diff
@@ -0,0 +1,95 @@
+
+
+from forecasting.datamodel.iris import *
+
+
+from forecasting.kgutils.javagateway import jpsBaseLibGW
+
+from forecasting.utils.useful_queries import get_covs_heat_supply
+from forecasting.datamodel.data_mapping import *
+
+import datetime as dt
+
+
+# PostgreSQL supported data types: https://www.jooq.org/javadoc/dev/org.jooq/org/jooq/impl/SQLDataType.html
+jpsBaseLibView = jpsBaseLibGW.createModuleView()
+Instant = jpsBaseLibView.java.time.Instant
+TIMECLASS = Instant.now().getClass()
+
+# Create data class for all time series data (i.e. all data as double)
+jpsBaseLibView = jpsBaseLibGW.createModuleView()
+DATACLASS = jpsBaseLibView.java.lang.Double.TYPE
+DOUBLE = jpsBaseLibView.java.lang.Double.TYPE
+INTEGER = jpsBaseLibView.java.lang.Integer.TYPE
+BOOLEAN = jpsBaseLibView.java.lang.Boolean.TYPE
+
+
+TIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+MODEL_MAPPING = {}
+""" KEYS in the MODEL_MAPPING dict:
+Required:
+'ts_data_type': Data type from jpsBaseLibView.java.lang
+'frequency': The frequency of the time series data
```

Comment:
What data type is expected here as the example uses `dt.timedelta(hours=1)`. Would string also work? Please clarify